itests/steps/marathon_steps.py

```diff
@@ -19,6 +19,11 @@ def working_marathon(context):
         context.client = marathon.MarathonClient(marathon_connection_string)
 
 
+@then(u'we get the marathon instance\'s info')
```

Comment:
I don't see this ever get called?
See `itests/marathon_python.feature` and start a new scenario for querying marathon in general